website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -0,0 +1,182 @@
+enabling traffic redirection.
+
+### VMs
+
+* For a service on a VM to be a part of the service mesh, it needs to run a Connect sidecar proxy.
+* The [`consul connect redirect-traffic`](/commands/connect/redirect-traffic) command needs to be run on the VM to
+set it up to redirect all inbound and outbound traffic to that VM through the sidecar proxy.
+* Services need to be registered with Consul.
+* If the default policy for ACLs is "deny", then Service Intentions should be set up to allow intended services to connect to each other.
+Otherwise, all Connect services can talk to all other services.
+
+
+## Configuration
+
+### Kubernetes
+
+Transparent proxy can be enabled in Kubernetes on the whole cluster via the Helm value:
+
+```yaml
+connectInject:
+  transparentProxy:
+    defaultEnabled: true
+```
+
+It can also be enabled on a per service basis via the annotation `consul.hashicorp.com/transparent-proxy=true` on the
+Pods for each service.
+
+### VMs
+
+In other environments, transparent proxy can be enabled via Proxy Defaults and Service Defaults config entries, or via
+the proxy service registration:
+```
+# Proxy defaults apply to all proxies.
+kind = "proxy-defaults"
+name = "global"
+
+mode = "transparent"
+transparent_proxy {
+  outbound_listener_port = 15001
+}
+```
+```
+# Service defaults apply to all instances of the web service.
+kind = "service-defaults"
+name = "web"
+
+mode = "transparent"
+transparent_proxy {
+  outbound_listener_port = 15001
+}
+```
+```
+# Proxy service registrations apply to a single proxy instance.
+name = "web-sidecar-proxy"
+kind = "connect-proxy"
+proxy {
+  mode = "transparent"
+  transparent_proxy {
+    outbound_listener_port = 15001
+  }
+  destination_service_name = "web"
+  local_service_port = 8080
+}
+port = 20000
+```
+
+Similar to `mesh_gateway.mode`, the new proxy mode will have the following string values:
+* "" - The empty string represents the default value for the feature, and allows for the mode to be overridden by
+central configuration, like “service-defaults”.
+* "direct" - Explicitly disables configuring transparent proxy, falling back to only configuring explicit upstreams.
+* "transparent" - Explicitly enables configuring transparent proxy.
+
+Additionally, the new Cluster config entry is scoped to the set of federated Consul datacenters and can be used to allow or block
+traffic to external destinations. This example shows blocking traffic to external destinations (outside of Consul's catalog):
+
+```
+kind = "cluster"
+name = "cluster"
+
+transparent_proxy {
+  catalog_destinations_only = true
+}
+```
+
+## Known Limitations
+
+* For services on VMs, transparent proxy is only supported if there is one service per VM. This is because the traffic
+redirection rules are applicable to the whole VM and are responsible for directing traffic from one service to it’s
+sidecar proxy.
```

Comment:
```suggestion
* For services on VMs, transparent proxy only supports one service per VM, or per network namespace. This is because the traffic
redirection rules are applicable to the entire namespace (incl. default namespace) and will directing all outbound traffic from the service to it’s sidecar proxy.
```